### PR TITLE
January CC Release Notes

### LINKED ISSUE #9360: January CC Release Notes
- Restore modal
- Fixed a bug where deleting your only organization will lock your email out of ever being able to be used again [jira](https://cockroachlabs.atlassian.net/browse/CC-3382)
- Fixed a bug where VPC peering appeared to be available on clusters that it wasn't supported on. [related pr](https://github.com/cockroachlabs/managed-service/pull/4459)

### PR DESCRIPTION
**Pull request by lnhsingh**
Closes #9360

### CODE REVIEW

**Comment by lnhsingh** [comment]
@emily-horing @kannanlakshmi @jramos-crl - can one you do a quick review of this month's release notes and lmk if things look good? Thank you!

**Review by kannanlakshmi — APPROVED** [review]
LGTM, thanks! I assume Free tier + PrivateLink will be part of the Feb release notes?

**Review by lnhsingh — COMMENTED** [review]
TFTR! Yeah, Free tier, PrivateLink, and restore will be in Feb!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9457#-:-MRebfxzC5GbxDEO28lu:b-9woy6r)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing, @jramos-crl, and @rmloveland)

File: releases/cockroachcloud-01222021.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{% include marketo.html %}
+
+### General Changes
+
+New CockroachCloud clusters will now run CockroachDB [v20.2.3](v20.2.3.html).
+
+### Bug Fixes
+
+- Fixed a bug where deleting your only organization would prevent your email from being used for a new organization in the future.
+- Fixed a bug where VPC peering appeared to be available on clusters that it wasn't supported on.
```
Comment (rmloveland) [inline releases/cockroachcloud-01222021.md:20]: Only feedback is that maybe "VPC peering" can be a link to some docs we have on VPC peering? (if we have)

**Review by rmloveland — APPROVED** [review]
LGTM with one comment you can ignore as needed

File: releases/cockroachcloud-01222021.md (hunk 1 above)
Comment (lnhsingh) [reply releases/cockroachcloud-01222021.md]: Done

**Pull request merged**